exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverExportUtils.java

```diff
@@ -106,6 +109,15 @@
   @VisibleForTesting static final String SUMMARY_SUFFIX_COUNT = "_summary_count";
   @VisibleForTesting static final String SUMMARY_SUFFIX_SUM = "_summary_sum";
 
+  @VisibleForTesting
+  static final String EXEMPLAR_ATTACHMENT_TYPE_STRING =
+      "type.googleapis.com/google.protobuf.StringValue";
+  // TODO: currently Stackdriver Monitoring only accepts string attachment.
```

Comment:
Not sure I understand this. I made a prototype some time ago and I was able to push the "SpanContext"

Reply:
Which type are you using?

I got this error with type `google.devtools.cloudtrace.v1.Trace`:
```json
{
  "code": 400,
  "message": "Invalid value at 'time_series[0].points[0].value.distribution_value.exemplars[0].attachments[0]' (Any), Invalid type URL, unknown type: google.devtools.cloudtrace.v1.Trace",
  "status": "INVALID_ARGUMENT",
}
```

Reply:
Use the proto that you added :) https://github.com/googleapis/googleapis/blob/master/google/monitoring/v3/span_context.proto

Reply:
I see - I tested the previous one because it's on Stackdriver documentation (https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries#exemplar):

> Contextual information about the example value. Examples are:
Trace ID: type.googleapis.com/google.devtools.cloudtrace.v1.Trace

Though it didn't work...

Reply:
Updated to support `SpanContext`, PTAL.